tests/unit/StaffNotation.spec.ts

```diff
@@ -293,11 +367,6 @@ describe('StaffNotation', () => {
         label: 'Record Conversion',
         disabled: false
       },
-      {
-        type: 'put-back-on',
-        label: 'Put Back On',
-        disabled: false
-      },
```

Comment:
Did you mean to restore this, too?

Reply:
For this one, I updated the unit test I've made specifically for this feature flag so that it would check if enabled/disabled on the menu items. It seemed redundant to still have this

Reply:
![image](https://github.com/bcgov/business-filings-ui/assets/17242795/ca26a504-c295-4cc1-8310-8900f0e830b7)

Reply:
In this case, I don't mind some redundancy for the sake of consistency 😃

Reply:
I see what you mean about the double find in your screnshot above. Feel free to update it as you see fit.